### PR TITLE
Dynamic overflow menu example

### PR DESCRIPTION
**Pull request by markewallace**
For demonstration purposes only

### CODE REVIEW

File: src/dialog/overflow-menu/overflow-menu.stories.ts [hunk 1]
```diff
@@ -1,7 +1,18 @@
 import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withKnobs, boolean, number } from "@storybook/addon-knobs/angular";
 
 import { DialogModule } from "../../";
 
+let options;
+
+const createOptions = (count: number): Array<string> => {
```
Comment (zvonimirfras) [inline src/dialog/overflow-menu/overflow-menu.stories.ts]: Should be a normal function if possible, to match other similar places in codebase.
```suggestion
const createOptions(count: number): Array<string> {
```

File: src/dialog/overflow-menu/overflow-menu.stories.ts [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+	if (options && count === options.length) {
+		return options;
+	}
+	options = Array(count).fill(0).map((x, i) => "Option " + (i+1));
+	return options;
+};
```
Comment (zvonimirfras) [inline src/dialog/overflow-menu/overflow-menu.stories.ts]:
```suggestion
}
```
to go with the function change.

File: src/dialog/overflow-menu/overflow-menu.stories.ts [hunk 3]
```diff
@@ -40,4 +52,18 @@ storiesOf("Overflow Menu", module)
 			</ibm-overflow-menu>
 			<ibm-dialog-placeholder></ibm-dialog-placeholder>
 		`
+	}))
+	.add("Dynamic", () => ({
+		template: `
+			<span>Dynamic OverflowMenu, using the optionCount knob to change the number of menu options</span>
```
Comment (zvonimirfras) [inline src/dialog/overflow-menu/overflow-menu.stories.ts]:
```suggestion
			<span>
				Dynamic <code>OverflowMenu</code>, using the <code>optionCount</code> knob
				to change the number of menu options
			</span>
```
to make code read as code

File: src/dialog/overflow-menu/overflow-menu.stories.ts [hunk 4]
```diff
@@ -1,7 +1,18 @@
...
 
 import { DialogModule } from "../../";
 
+let options;
+
+const createOptions = (count: number): Array<string> => {
+	if (options && count === options.length) {
+		return options;
+	}
+	options = Array(count).fill(0).map((x, i) => "Option " + (i+1));
```
Comment (zvonimirfras) [inline src/dialog/overflow-menu/overflow-menu.stories.ts]:
```suggestion
	options = Array(count).fill(0).map((x, i) => "Option " + (i + 1));
```

**Comment by zvonimirfras** [comment]
There are also some conflicts.

File: src/dialog/overflow-menu/overflow-menu.stories.ts (hunk 4 above)
Comment (zvonimirfras) [reply src/dialog/overflow-menu/overflow-menu.stories.ts]: Is it just my github or is this still pending?

**Comment by markewallace** [comment]
@zvonimirfras that's fixed now

**Pull request merged**